### DISCUSSION TITLE
eme_filterform - drop-down menu not showing

### DISCUSSION THREAD

**Discussion by wihast** [liedekef/events-made-easy#426] [General]
After an update (dont´t know which version i had before) to Version 2.4.25 the [eme_filterform] that i configured in the widget isn´t working correctly. It doesn´t create the drop-down menu. 

When i load the site it shows for like a second the drop-down menu and then only a "searchfield" is displayed and when i click into the field i get the categories. 

I have set [eme_filterform multiple=1 template_id=3 multisize=10 submit=Suchen] in the widgetarea.
The template_id=3 contains "#_FILTER_CATS"
It worked before the update. 

Now the strange thing:
After i set [eme_filterform multiple=1 template_id=3 multisize=10 submit=Suchen] twice in the widgetarea ([eme_filterform multiple=1 template_id=3 multisize=10 submit=Suchen] [eme_filterform multiple=1 template_id=3 multisize=10 submit=Suchen]) the first shortcode gives me the dropdown menu back (number 1 in image) and the second shortcode behaves like I described above (number 2 in image).

![eme_filterform](https://github.com/liedekef/events-made-easy/assets/157857187/120288f5-7002-4e71-ae73-ca11fedb813b)

**Comment by liedekef** [comment]
The dropdown has indeed become a "smarter" dropdown that allows a search (which wasn't the case before).
In fact the second example you mention is not supported: two filterforms will cause issues ... (in fact because the name of the form is not made unique ==> I'll fix that for the next release).

**Comment by wihast** [comment]
How can i get back the old dropdown? I need it like in number 1. Must i downgrade for that?

**Reply by liedekef** [reply]
What's the difference? When clicking in the field, you get the dropdown as well, so there's no difference.
Or is something wrong with the way things work now?

**Comment by wihast** [comment]
The difference is that you can see before which categories are existing. Seems relevant for my sites because it´s very helpful for people that they see can see from categories they can choose.
Is there are a possibility to implement a switch to turn that back?

**Reply by liedekef** [reply]
The next version will have a new option (old_select) for the filter form, so you can have it back.
Also: I saw the label was not being shown correctly currently in your situation: that's fixed too (so maybe that is sufficient in itself ...)
See https://github.com/liedekef/events-made-easy/commit/f635fbe7a03c17b74daffe7bcd196a407e5247e5

**Reply by wihast** [reply]
Thanks a lot for that. 👍

**Comment by wihast** [comment]
Close as resolved.
